components/avatar/avatar.vue

```diff
@@ -309,6 +337,19 @@ export default {
     showImage () {
       return this.imageLoadedSuccessfully !== false && this.imageSrc;
     },
+
+    avatarListeners () {
+      if (!this.clickable) return {};
+
+      return {
+        click: (e) => this.$emit('click', e),
+        keydown: (e) => {
+          if (!['Space', 'Enter'].includes(e.code)) return;
+          e.preventDefault();
+          this.$emit('click', e);
+        },
```

Comment:
I think we wouldn't have to worry about all this if the element was an actual button

Reply:
I'll try with a button